packages/conf-openblas/conf-openblas.0.1/opam

```diff
@@ -11,7 +11,7 @@ build: [
   ["sh" "-exc" "cc $CFLAGS test.c -lopenblas"] {os = "ubuntu" }
   ["sh" "-exc" "cc $CFLAGS test.c -lopenblas"] {os = "debian" }
   ["sh" "-exc" "cc $CFLAGS test.c -L/opt/OpenBLAS/lib -lopenblas"] {os = "centos"}
-  ["sh" "-exc" "cc $CFLAGS test.c -L/usr/local/opt/openblas/lib -lopenblas"] {os = "darwin"}
+  ["sh" "-exc" "cc $CFLAGS -I/usr/local/opt/openblas/include test.c -L/usr/local/opt/openblas/lib -lopenblas"] {os = "darwin"}
```

Comment:
it's likely that this only works on macports and not homebrew -- or does it work on both?